homeassistant/components/recorder/__init__.py

```diff
@@ -581,9 +581,9 @@ def _async_setup_periodic_tasks(self):
             self.hass, self.async_nightly_tasks, hour=4, minute=12, second=0
         )
 
-        # Compile hourly statistics every hour at *:12
+        # Compile short term statistics every 5 minutes
         async_track_time_change(
-            self.hass, self.async_hourly_statistics, minute=12, second=0
+            self.hass, self.async_periodic_statistics, minute=range(3, 60, 5), second=0
```

Comment:
Why 3 ?

Reply:
This is the time where `async_periodic_statistics` will be called: 03, 08, 13, .. 58
The periods will still be 00~05, 05~10, .. 55~00.

I just don't want to have edge cases where the statistics run misses some state changes, but maybe that can't happen?

Reply:
I don't think it can happen because this injects a task and the recorder processes tasks FIFO

Reply:
Changed to a 10s offset as discussed on Discord